extensions/amp-analytics/0.1/vendors/blueconic.json

```diff
@@ -0,0 +1,15 @@
+{
+  "requests": {
+    "event": "https://${tenantId}/s/${pixelId}.gif?${trackingIdentifier}${defaultTrackingValues}${extraTrackingValues}"
+  },
+  "vars": {
+    "trackingIdentifier": "property=${property}&${property}=${clientId(cid-scope-cookie-fallback-name)}",
+    "defaultTrackingValues": "&sourceUrl=${sourceUrl}&canonicalUrl=${canonicalUrl}&referrer=${documentReferrer}&title=${title}&visitDate=${timestamp}&geoLocation=${ampGeo}&userAgent=${userAgent}",
+    "extraTrackingValues": ""
+  },
+  "transport": {
+    "beacon": false,
+    "xhrpost": false,
+    "image": true
+  }
```

Comment:
AMP handles the fallback of transport options (beacon > xhrpost > image). We recommend you use the default transport settings (by omitting the transport object) unless your system can only handle image requests.